File: usedesk_sdk/src/main/java/ru/usedesk/sdk/data/framework/api/entity/request/SendMessageRequest.java
package ru.usedesk.sdk.data.framework.api.entity.request;

public class SendMessageRequest extends BaseRequest {

    public static final String TYPE = "@@server/chat/SEND_MESSAGE";

    private RequestMessage message;

    public SendMessageRequest(String token, RequestMessage requestMessage) {
        super(TYPE, token);
        this.message = requestMessage;
    }

    public RequestMessage getRequestMessage() {
        return message;
    }
}

File: chat-sdk/src/main/java/ru/usedesk/chat_sdk/data/repository/api/_entity/request/SendMessageRequest.kt
package ru.usedesk.chat_sdk.data.repository.api._entity.request

import ru.usedesk.chat_sdk.data._entity.UsedeskFile

class SendMessageRequest(
        token: String,
        text: String? = " ",
        usedeskFile: UsedeskFile? = null
) : BaseRequest(TYPE, token) {

    private val message = RequestMessage(text, usedeskFile)

    companion object {
        private const val TYPE = "@@server/chat/SEND_MESSAGE"
    }

    private class RequestMessage(
            private val text: String?,
            private val usedeskFile: UsedeskFile?
    )
}